backend/itrabaho/models.py

```diff
@@ -28,6 +28,7 @@ class UserModel(AbstractBaseUser, PermissionsMixin):
     phoneNumber = models.CharField(max_length=MOBILE_NUMBER_MAX_LENGTH, unique=True)
     sex = models.CharField(max_length=1, choices=SexChoices.choices)
     birthDate = models.DateField(blank=True, null=True)
+    lastLogin = models.DateField(auto_now_add=True)
```

Comment:
setting `DateField` to `auto_now_add=True` is non-editable

```suggestion
    lastLogin = models.DateField(null=True, blank=True)
```